cli/cli_app.go

```diff
@@ -61,6 +62,14 @@ func CreateTerragruntCli(version string) *cli.App {
    Moreover, for the apply and destroy commands, Terragrunt will first try to acquire a lock using DynamoDB. For
    documentation, see https://github.com/gruntwork-io/terragrunt/.`
 
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "config",
+			Value: os.Getenv("TERRAGRUNT_CONFIG"),
```

Comment:
My project needs support for many regions, also working directly on the cli executing commands like 
`
terraform apply
terraform plan
`

Makes nosense when your project starts getting multi-regional, so you can have something like:
`
$ tree -L 1
|-- Makefile
|-- README.md
|-- Rakefile
|-- Terrafile
|-- aws_region (each folder with its own environment_vars.tf file)
   |_ ireland
   |_ virginia
   |_ oregon
   |_ ........
|-- envs
   |_ bastion.tf
   |_ puppetmaster.tf
   |_ elasticsearch.tf
|-- global_variables.tf
`

And you can have a makefile that builds your infra depending on your ENV:

Makefile:
`
TF=terraform
# TF=terragrunt

TF_ENV ?= ireland
AWS_REGION ?= eu-west-1
TF_VAR_aws_account_id=`aws sts get-caller-identity --output text | cut -f1`

TF_VARS=TF_VAR_aws_account_id=$(TF_VAR_aws_account_id)
TF_ARGS=aws_region/$(TF_ENV)

apply:
	$(TF_VARS) $(TF) apply $(TF_ARGS)
`